draft-ietf-mops-streaming-opcons.md

```diff
@@ -325,7 +325,7 @@ This document examines networking and transport protocol issues as they relate t
 
 This document specifically focuses on streaming applications and defines streaming as follows:
 
-- Streaming is the transmission of continuous media from a server to a client and its simultaneous consumption by the client.
+- Streaming is the continuous transmission of media from a server to a client and its simultaneous consumption by the client.
 
 - Here, "continuous media" refers to media and associated streams such as video, audio, metadata, etc. In this definition, the critical term is "simultaneous," as it is not considered streaming if one downloads a media file and plays it after the download is completed, which would be called download-and-play.
```

Comment:
Here, "media" refers to any type of media and associated streams such as video, audio, metadata, etc.

Reply:
I agree - will make this change and merge.